modules/operator-georepl-site-removal.adoc

```diff
@@ -0,0 +1,59 @@
+:_content-type: PROCEDURE
+[id="operator-georepl-site-removal"]
+= Removing a geo-replicated site from your {productname} Operator deployment
+
+By using the following procedure, {productname} administrators can remove sites in a geo-replicated setup. 
+
+.Prerequisites 
+
+* You are logged into {ocp}.
+* You have configured {productname} geo-replication with at least two sites, for example, `usstorage` and `eustorage`. 
+* Each site has its own Organization, Repository, and image tags. 
```

Comment:
This description is not correct, actually only the imagestorageplacement is associated with storagelocation.